### ISSUE TITLE
openapi-schema-validation has been deprecated

### ISSUE THREAD

**Issue by stevenwjy** [anttiviljami/openapi-backend#98]
According to the description [here](https://www.npmjs.com/package/openapi-schema-validation), the validator that is used in this library to perform schema validation has been deprecated. It would be great to change the dependencies to use its replacement, which is `openapi-schema-validator`.

**Comment by snakuzzo** [comment]
Should be related to my issue...???

`Error: schema is invalid: data.properties['path'].properties['numLinea'].exclusiveMaximum should be number`

```
          {
            "name": "myparameter",
            "in": "path",
            "description": "my parameter",
            "required": true,
            "schema": {
              "type": "number",
              "minimum": 1,
              "maximum": 20,
              "exclusiveMinimum": false,
              "exclusiveMaximum": true
            }
          }
```

**Comment by snakuzzo** [comment]
Hi @anttiviljami ... any news about this ?

**Comment by anttiviljami** [comment]
@tenhaus already implemented this for openapi-client-axios. I'll just port his code over :) https://github.com/anttiviljami/openapi-client-axios/pull/43

**Comment by anttiviljami** [comment]
Fixed in `openapi-backend@3.6.3`
